src/token/erc20/erc20.cairo

```diff
@@ -12,11 +12,11 @@ mod ERC20 {
 
     #[storage]
     struct Storage {
-        _name: felt252,
-        _symbol: felt252,
-        _total_supply: u256,
-        _balances: LegacyMap<ContractAddress, u256>,
-        _allowances: LegacyMap<(ContractAddress, ContractAddress), u256>,
+        ERC20_name: felt252,
+        ERC20_symbol: felt252,
+        ERC20_total_supply: u256,
```

Comment:
hm this means that if there's a non-`18`, storage-based `decimals` out there, this would result in a very dangerous upgrade.

+1, this has to be clearly explained and referenced in the upgrades and erc20 sections of the doc

Reply:
Should we document them for the release?